src/controller/signup.controller.ts

```diff
@@ -8,6 +8,11 @@ export default {
       const id = await serviceSignup.createUser(req.body);
       res.status(201).send(id);
     } catch (error) {
+      let message;
+      if (error instanceof Error) {
+        message = error;
+        res.status(400).send(message.toString());
```

Comment:
I see that most of our errors here are just strings, but we should be careful what we are sending back to the client. Maybe in this case we log the error in the backend and send a generic message to the front?